lld/COFF/Driver.h

```diff
@@ -86,8 +86,8 @@ class LinkerDriver {
 
   void addClangLibSearchPaths(const std::string &argv0);
 
-  // Used by the resolver to parse .drectve section contents.
-  void parseDirectives(InputFile *file);
+  // Used by the resolver to iterate through .drectve section(s).
+  void processDirectivesSection(InputFile *file);
```

Comment:
I am still a bit unhappy with the naming. I wonder if we shouldn't just leave `parseDirectives` for both functions.